src/components/picture-in-picture/picture-in-picture.js

```diff
@@ -87,13 +92,14 @@ class PictureInPicture extends Component {
     if (this.props.isPictureInPictureSupported && this.props.playerSize !== PLAYER_SIZE.EXTRA_SMALL) {
       return (
         <div className={[style.controlButtonContainer, style.pictureInPicture].join(' ')}>
-          <Tooltip label={this.props.pipText}>
+          <Tooltip label={this.props.isInPictureInPicture ? this.props.pipOpenTooltipText : this.props.pipCloseText}>
```

Comment:
small one, but why one has tooltip in it and other doesn't?

Reply:
I didn't want to change existing keys. The existing one is used for both tooltip and aria label